presentation/src/main/res/values/colors.xml

```diff
@@ -3,5 +3,5 @@
     <color name="colorPrimary">#F44336</color>
     <color name="colorPrimaryDark">#D32F2F</color>
     <color name="colorAccent">#D81B60</color>
-    <color name="colorWhite">#FFFFFF</color>
+    <color name="black_baground">#292929</color>
```

Comment:
This color is not really black, use a tool like [this one](http://chir.ag/projects/name-that-color/#292929) to give it a better name

Reply:
Esta chevere esa herramienta para saber el color

Reply:
@SalosV please try to answer in English, that way you'll practice.